### DISCUSSION TITLE
Keep logged in ?

### DISCUSSION THREAD

**Discussion by amartini** [cockpit-project/cockpit#17792] [Q&A] [answered]
Is there an option to keep you logged in for a long time, so I dont have to input credentials?
I have set [Session] IdleTimeout = 0 on the conf file, but I'm still logged out after some time.

**Comment by marusak** [comment]
setting to 0 or just removing this option will never log you out. You need to restart cockpit. Ideally just restart cockpit systemd unit.

**Comment by amartini** [comment]
What happens is if I close the tab, it log me out. Is there a way to keep logged in even after closing the tab? (Like most sites do)

**Reply by martinpitt** [reply, marked as answer]
Yes, there is no way to configure cockpit to not log you out after closing the tab. Unlike most sites, a Cockpit session takes significant resources: There's a full Linux user session running, and you need *some* kind of connection (i.e. file descriptor/pipe) to its running process. That's not something which you can "store" in a browser. The alternative would be to close the session, but skip authentication the next time via some PAM module -- but that would be prohibitively dangerous.
